### PR TITLE
Development

### PR DESCRIPTION
**Pull request by i4seeu**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,166 @@
+* {
```
Comment (oluwoleilesanmi) [inline assets/css/style.css]: Use reset.css file for reseting propreties.

**Review by oluwoleilesanmi — CHANGES_REQUESTED** [review]
@kudjieRaymond @i4seeu Well done good use of relative units of measure %, good use of flex box.make these changes and you will be good to go.

- Use a reset.css file in project.
- Use [HTML validator](https://www.freeformatter.com/html-validator.html) to check and improve your markup.
- Use [Css Lint](csslint.net) to improve your css. you will learn even better practices if you use these tools.

File: assets/css/style.css (hunk 1 above)
Comment (kudjieRaymond) [reply assets/css/style.css]: oluwoleilesanmi,  The Css lint shows warnings not error.. Do you have any suggestions as to  how we can resolve them?

**Review by oluwoleilesanmi — APPROVED** [review]
@kudjieRaymond @codermlw Good work guys :+1:

File: assets/css/style.css (hunk 1 above)
Comment (oluwoleilesanmi) [reply assets/css/style.css]: - You can ignore warnings, and you may not be able to clear all.
- Just keep some of those warnings in mind, they will be helpful in writing better css later.
- Well done team :+1:

**Pull request merged**